src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardPlayerRequestResolver.java

```diff
@@ -115,23 +115,24 @@ public void resolve(@NotNull final IRequestManager manager, @NotNull final IRequ
             final EntityPlayer owner = ServerUtils.getPlayerFromUUID(colony.getWorld(), ((Colony) colony).getPermissions().getOwner());
             final TextComponentString colonyDescription = new TextComponentString(colony.getName() + ":");
 
-            if (owner != null)
+            if (manager.getColony().getWorld().isDaytime())
```

Comment:
NO!. This means that during the night this goes wrong. REVERT!

Reply:
it will still execute it just will stop requesting to the player during the night since we got a lot of complaints of players about workers requesting while they're asleep.

Reply:
HE!?!?!?!?!?!? But okey.

Reply:
Yeah players got super confused by them requesting at night and since they don't know where they work. At the same time the entities probably don't detect accordingly when you deliver the items at night.